README.md

```diff
@@ -1,42 +1,211 @@
-# mi-debian
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
 
-This repo contains live-build files and scripts to create an custom Debian 7 
-ISO image with the necessary packages and sdc-vmtools installed for deploying 
-on SmartOS and Joyent Public Cloud.
+<!--
+    Copyright 2020 Joyent, Inc.
+-->
 
-For more information on live-build see the [live-build manual](http://live-systems.org/manual/stable/html/live-manual.en.html)
+# Debian Image Builder
+
+This repo allows one to create custom Debian images for use with Triton.
```

Comment:
Should there be cross-referencing that Centos (and others?) should see the mi-centos-hvm repo?

Reply:
Added link to github search.